src/explore.sh

```diff
@@ -210,10 +227,11 @@ function explore_files_under_git()
   local regex="$1"
   local path="$2"
   local flag="$3"
+  local context="$4"
 
   flag=${flag:-'SILENT'}
 
-  cmd_manager "$flag" "git grep -e '$regex' -nI $path"
+  cmd_manager "$flag" "git grep -C $context -e '$regex' -nI $path"
```

Comment:
Yes, I actually wanted to confirm this. I understand now.
Similarly, if I understand correctly, I don't think we are following [this rule](https://kworkflow.org/content/codingstyle.html#avoid-short-options) of the coding style which suggests using long options instead of short ones, wherever available. 

According to it the command should look like - `git grep --context ${context} -e '${regex}' --line-number -I ${path}`

Reply:
Yes, I actually wanted to confirm this. I understand now.
Similarly, if I understand correctly, I don't think we are following [this rule](https://kworkflow.org/content/codingstyle.html#avoid-short-options) of the coding style which suggests using long options instead of short ones, wherever available. 

According to it the command should look like - `git grep --context ${context} -e '${regex}' --line-number -I ${path}`

Reply:
Nice catch! You are correct, this rule should also be enforced, so take the opportunity to do this here (and in the other two functions).

Also, just as a reminder, you probably will find many other places in `src/explore.sh` that don't follow the coding style, but **don't change them** in this PR, as they would be outside the scope of the change.